go/enclave/enclave.go

```diff
@@ -227,20 +228,35 @@ func (e *enclaveImpl) SubmitRollup(rollup common.ExtRollup) {
 	}
 }
 
-func (e *enclaveImpl) SubmitTx(tx common.EncryptedTx) error {
+func (e *enclaveImpl) SubmitTx(tx common.EncryptedTx) (common.EncryptedResponseSendRawTx, error) {
```

Comment:
As part of the `eth_sendRawTransaction` API, we must return a tx hash, hence the return type here.